core/App/ffi_operation.cpp

```diff
@@ -1432,7 +1432,7 @@ extern "C"
         }
         appid->datalen = UUID_STR_LEN;
```

Comment:
commit msg
"
Use calloc to zero the memory of apikey buffer

test: passed the unit test
Signed-off-by: luoyiding <yidingx.luo@intel.com>
"

Reply:
done